schema.sql

```diff
@@ -51,3 +51,35 @@ ALTER TABLE animals
 	ON DELETE CASCADE;	
 
 select * from animals
+
+/*Day Four*/
```

Comment:
- [ ] Dear @JOdiemo, Please move these and all other select statements to the `queries.sql` file. Remember, everything that is a query that helps to analyze data should be saved in the `queries.sql` file.